azure/ConsiderationsForServiceDesign.md

```diff
@@ -141,94 +142,196 @@ cannot collide with a resource path that contains user-specified resource ids.
-operations.
+The operation is initiated with a POST operation and the operation path ends in `:action`.
 
-Note: The RELO pattern should not be used in cases where the completion status of individual operations
-may be important to users, as opposed to simply learning that an operation of the type they requested
-(e.g. create a resource with a specific name) has successfully completed.
+```text
+POST /<service-or-resource-url>:action 
+Operation-Id: 22 
+ 
+{ 
+   "arg1": 123 
+   "arg2": "abc" 
+} 
+```
 
-### Long-running operations with status monitor
+The response is a `202 Accepted` as described above.
+
+```text
+HTTP/1.1 202 Accepted
+Operation-Location: https://<status-monitor-endpoint>/22
+Retry-After: 5 
+ 
+{
+   "id": "22",
+   "status": "NotStarted"
+}
+```
 
-In the LRO with status monitor pattern, the status and results of the operation are encapsulated into
-a status monitor resource that is distinct from the target resource and specific to the individual
-operation request.  Here's what the status monitor LRO pattern looks like:
+The client will issue a GET to the status monitor to obtain the status and result of the operation.
 
-<!-- markdownlint-disable MD033 -->
-<p align="center">
-  <img src="./statmon.jpg" alt="The status monitor LRO flow"/>
-</p>
-<!-- markdownlint-enable MD033 -->
+```text
+GET https://<status-monitor-endpoint>/22?api-version=2022-05-01
+```
 
-1. The client sends the request to initiate the long-running operation.
-As in the RELO pattern, the initial request could be a PUT, PATCH, POST, or DELETE method.
+When the operation completes successfully, the result (if there is one) will be included in the `result` field of the status monitor.
 
-2. The resource validates the request and initiates the operation processing.
-It sends a response to the client with a `202-Accepted` HTTP status code.
-Included in this response is an `Operation-location` response header with the absolute URL of
-status monitor for this specific operation.
-The response also includes a `Retry-after` header telling the client a minimum time to wait (in seconds)
-before sending a request to the status monitor URL.
+```text
+HTTP/1.1 200 Ok
+ 
+{
+   "id": "22",
+   "status": "Succeeded",
+   "result": { ... }
+}
+```
 
-3. After waiting at least the amount of time specified by the previous response's `Retry-after` header,
-the client issues a GET request to the status monitor URL.
+### Create (PUT) with additional long-running processing
 
-4. The status monitor URL responds with information about the operation including its current status,
-which should be represented as one of a fixed set of string values in a field named `status`.
-If the operation is still being processed, the status field will contain a "non-terminal" value, like `Processing`.
+A special case of long-running operation that occurs often is a PUT operation to create a resource
```

Comment:
We should give an example of this to make this a bit more clear.

Reply:
Added.